contracts/package.json

```diff
@@ -32,22 +32,23 @@
     "postbuild": "cp -r ./artifacts ./build",
     "types": "tsc -p tsconfig.json --noEmit",
     "docs": "hardhat docgen",
-    "test": "hardhat test --network hardhat",
-    "test:maci": "pnpm run test ./tests/MACI.test.ts",
-    "test:poll": "pnpm run test ./tests/Poll.test.ts",
-    "test:messageProcessor": "pnpm run test ./tests/MessageProcessor.test.ts",
-    "test:tally": "pnpm run test ./tests/Tally.test.ts",
-    "test:hasher": "pnpm run test ./tests/Hasher.test.ts",
-    "test:utilities": "pnpm run test ./tests/Utilities.test.ts",
-    "test:signupGatekeeper": "pnpm run test ./tests/SignUpGatekeeper.test.ts",
-    "test:verifier": "pnpm run test ./tests/Verifier.test.ts",
-    "test:accQueue": "pnpm run test ./tests/AccQueue.test.ts",
-    "test:accQueueBenchmark": "pnpm run test ./tests/AccQueueBenchmark.test.ts",
-    "test:hasherBenchmarks": "pnpm run test ./tests/HasherBenchmarks.test.ts",
-    "test:vkRegistry": "pnpm run test ./tests/VkRegistry.test.ts",
-    "test:pollFactory": "pnpm run test ./tests/PollFactory.test.ts",
-    "test:subsidy": "pnpm run test ./tests/Subsidy.test.ts",
-    "test:eas_gatekeeper": "pnpm run test ./tests/EASGatekeeper.test.ts",
+    "test": "hardhat test",
```

Comment:
could we please have these back as they were (+ the new test command ofc). We don't have hardhat as default network in the config, so tests will run against the wrong network this way

Reply:
huh, not sure what casued these to change, maybe somehow the rebase?

I'll change them back

Reply:
Seems like `--network hardhat` was removed from `test`, can you add it back pls? and this way we can remove from `test:hatsGatekeeper` -> which would be just `pnpm run test ./tests/HatsGatekeeper.test.ts`

Reply:
good catch, thanks